officemanager/setup.py

```diff
@@ -0,0 +1,31 @@
+import os
+from setuptools import find_packages, setup
+
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as readme:
+    README = readme.read()
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='django-office-manager',
+    version='0.1',
+    packages=find_packages(),
+    include_package_data=True,
+    license='MIT License',
+    description='A simple Django app to manage departments and employees.',
+    long_description=README,
+    author='Dmitri Zawadski',
+    author_email='dzdimati@gmail.com',
+    classifiers=[
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Framework :: Django :: 2.2',
+        'License :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3.6',
+        'Topic :: Internet :: WWW/HTTP',
+        'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
+    ],
```

Comment:
Install_requires is missing. Please add install_requires with all required dependencies (django, DRF, etc).